tests/foreman/ui/test_host.py

```diff
@@ -865,6 +865,39 @@ def test_positive_search_by_parameter_with_operator(self):
             self.assertIsNotNone(self.hosts.wait_until_element(
                 (strategy, value % additional_host.name)))
 
+    @run_only_on('sat')
+    @tier2
+    def test_positive_search_with_org_and_loc_context(self):
+        """Perform usual search for host, but organization and location used
+        for host create procedure should have 'All capsules' checkbox selected
+
+        :id: 2ce50df0-2b30-42cc-a40b-0e1f4fde3c6f
+
+        :expectedresults: Search functionality works as expected and correct
+            result is returned
+
+        :BZ: 1405496
+
+        :CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        loc = entities.Location().create()
+        host = entities.Host(organization=org, location=loc).create()
+        with Session(self.browser) as session:
+            self.org.search_and_click(org.name)
+            self.org.click(tab_locators['context.tab_capsules'])
+            self.org.assign_value(locators['org.all_capsules'], True)
+            self.org.click(common_locators['submit'])
+            self.location.search_and_click(loc.name)
+            self.location.click(tab_locators['context.tab_capsules'])
+            self.location.assign_value(locators['location.all_capsules'], True)
+            self.location.click(common_locators['submit'])
```

Comment:
Because here we talk about checkbox rather than usual select lists, so it will be completely out of style for `_configure_org` . Also it will be necessary to expand parameters list which is already ugly at that moment. Should I implement that?